azure-pipelines-os-integration.yml

```diff
@@ -165,18 +158,12 @@ stages:
             displayName: 'Configure mysql'
 
           - script: |
-              pip install \
-                --requirement reqs/requirements-dev-test.txt \
-                --requirement reqs/requirements-dev-sqlalchemy.txt
+              pip install ".[test, mysql]" pytest-azurepipelines
```

Comment:
Only installing `mysql` test instead of installing everything in `requirements-dev-sqlalchemy`. love it